tensorrt_llm/_torch/auto_deploy/transformations/library/fusion.py

```diff
@@ -76,13 +96,18 @@ def split_output(tensor: torch.Tensor) -> Tuple[torch.Tensor, ...]:
     else:
         param_fused = nn.Parameter(fuse_weights([gm.get_parameter(k) for k in keys_unfused]))
 
-    setattr(gm, key_fused, param_fused)
+    weight_key = keys_unfused[0]
+    weight_module_path, _ = weight_key.rsplit(".", 1)
+    new_module_name = f"{weight_module_path}_fused"
+    full_new_param_name = add_new_parameter_to_submodule(
+        gm, new_module_name, key_fused, param_fused, ad_logger
+    )
 
     # Handle fused_kwargs for quantized fused gemm.
     fused_kwargs = dict(linear_nodes[0].kwargs)
 
     with gm.graph.inserting_before(linear_nodes[0]):
-        get_param_node = gm.graph.get_attr(key_fused, torch.Tensor)
+        get_param_node = gm.graph.create_node("get_attr", full_new_param_name)
```

Comment:
what's wrong with
```suggestion
        get_param_node = gm.get_attr(full_new_param_name, torch.Tensor)
```
?

Reply:
let's use `gm.get_attr`. If there is a utility by torch.fx supporting this functionality, why not use it :)